validator/validator.proto

```diff
@@ -380,7 +380,7 @@ message AmpLayout {
   repeated Layout supported_layouts = 1;
   // Specifies whether this element has default dimensions. This corresponds
   // to being allowlisted in naturalDimensions_ in
-  // https://github.com/ampproject/amphtml/blob/main/src/layout.js.
+  // https://github.com/ampproject/amphtml/blob/main/src/static-layout.js.
```

Comment:
This one is not like the others. `static-` vs `core/dom/`

Reply:
It's referencing [`naturalDimensions_`](https://github.com/ampproject/amphtml/blob/main/src/static-layout.js#L54) which lives there now. The original `layout.js` was split, most of it living in `#core` while a small chunk relating to static layout remained in `src/static-layout.js`